### PR TITLE
Add a requirements.txt to the ansible container and have pip install …

### PR DESCRIPTION
**Pull request by sdwenham**
…netaddr for docker swarm (using the ipaddr filter) and the openstack sdk to allow interaction with openstack

### CODE REVIEW

**Comment by sdwenham** [comment]
I don't think I blew up the CI job as I didn't touch the alpine containers

**Comment by rjw57** [comment]
> I don't think I blew up the CI job as I didn't touch the alpine containers

I think builds have been failing for a couple of days now given the emails from CircleCI

**Comment by rjw57** [comment]
I've opened #12 which I think should fix the unrelated problems.

File: ansible-playbook/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+openstacksdk
```
Comment (rjw57) [inline ansible-playbook/requirements.txt:3]: Could we have a comment explaining which ansible features these are required for?

**Review by rjw57 — CHANGES_REQUESTED** [review]
Unsurprising for me, I know, but I'd like some more docs...

File: ansible-playbook/requirements.txt (hunk 1 above)
Comment (sdwenham) [reply ansible-playbook/requirements.txt:3]: This is an enabler for all the OpenStack modules at
https://docs.ansible.com/ansible/latest/modules/list_of_cloud_modules.html#openstack
I have been using os_server to create VMs. Where did you want the docs?

Comment (rjw57) [reply ansible-playbook/requirements.txt:3]: A comment above the requirements would be grand.

**Review by rjw57 — APPROVED** [review]

**Comment by rjw57** [comment]
Will merge if checks pass

**Pull request merged**